config/__init__.py

```diff
@@ -22,9 +22,18 @@
 VERSION = f"{SHORT_VERSION}{VERSION_DEV_PHASE}{VERSION_PHASE}"
 LONG_VERSION = f"{SHORT_VERSION}{PATCH_VERSION}{VERSION_DEV_PHASE}{VERSION_PHASE}"
 
+# tentacles
+# TODO: use real link
+DEFAULT_TENTACLES_URL = f"https://raw.githubusercontent.com/GuillaumeDSM/test_tentacles2/master/tentacles.zip"
```

Comment:
Should we use `www.tentacles.octobot.online` here ?

Reply:
yes, when it will be available (hopefully tonight)